addons/kommander/1.163.x/kommander-1.yaml

```diff
@@ -51,12 +51,7 @@ spec:
         konvoy:
           allowUnofficialReleases: true
         kubeaddonsRepository:
```

Comment:
You can safely remove this whole section actually and just stick with the chart defaults.

Reply:
Honestly I wanted to be explicit, but if that is bad, then I would remove it. I didn't find it to be a bad decision.